### DISCUSSION TITLE
deductibility_vat_percentage new error

### DISCUSSION THREAD

**Discussion by aster94** [fattureincloud/api#450] [Q&A] [answered]
Hello,

Since today I got a new validation error, I wasn't getting it before: 
`{"error":{"message":"Invalid request.","validation_result":{"data.items_list.0.deductibility_vat_percentage":["The data.items list.0.deductibility vat percentage must be a number."]}}}`
also in the gui i see that all my expenses are getting uploaded with a mark 
![image](https://github.com/fattureincloud/api/assets/21184718/202c9ced-7825-43ad-9a45-21208c4b6dc2)

is this something you may resolve or we should find a solution on our own?

**Comment by valmoz** [comment, marked as answer]
Hi @aster94 
Unfortunately, I won't be able to do a lot without your company ID. 
Reading the error message, I suppose you're sending a string instead of a number in the suggested field for the first item (maybe you added the € character?).
I suggest that you contact our Customer Support, attaching the link to this discussion, so that we can obtain your Company ID and proceed with a detailed analysis.

Thanks
Mauro

**Comment by aster94** [comment]
the problem was not from our side but from fatture in cloud, they solved the problem

**Reply by valmoz** [reply]
Hi @aster94 
we're glad that the problem is now solved.

Thanks
Mauro
